diagnostic.md

```diff
@@ -71,13 +73,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
 
  <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+Since the command 'cd planning' does not contain the full file path (i.e. /Users/nari/project/planning), we know this is a relative path. A path is relative if directions to get to the destination folder are relative to the current location.
+<!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
-
+I would probably use an absolute path when referring to images. If the file structure changes, it might be easier to correct that file path in this format as opposed to changing the rest of the code to reflect the new parent directory. 
```

Comment:
In this case relative would probably be easier since the image file lives inside our current working directory, project. As a result, an absolute path is not needed to access it.